http.go

```diff
@@ -170,6 +174,13 @@ func probeHTTP(target string, w http.ResponseWriter, module Module) (success boo
 			success = true
 		}
 
+		// If the response was gzipped and was transparently uncompressed
+		if resp.Uncompressed {
+			content_compressed = 1
+			if body, err := ioutil.ReadAll(resp.Body); err == nil {
+				content_length = int64(len(body))
```

Comment:
So the removal of the header sounds like a bug then, though I can see how it would lead to confusion for some as it's about the frame rather than the packet.